docs/how-to/native-install/post-install.rst

```diff
@@ -25,7 +25,7 @@ Post-installation instructions
     .. code-block:: bash
         :substitutions:
 
-        export PATH=$PATH:/opt/rocm-|rocm_version|/bin
+        export PATH=$PATH:/opt/rocm-|rocm_directory_version|/bin
```

Comment:
These substitutions are not taking effect.  There's probably something missing in conf.py like (line 60-62).

Like: ".. |amdgpu_version| replace:: {amdgpu_version}", but for "rocm_directory_version"

Reply:
I made the change and it seems to be working now.